app/src/main/java/com/pvp/app/ui/screen/settings/SettingsScreen.kt

```diff
@@ -173,6 +175,118 @@ private fun SettingHydrationNotificationToggle(
     )
 }
 
+@Composable
+private fun SettingApplicationTheme(
+    model: SettingsViewModel = hiltViewModel()
+) {
+    val themeValue by model
+        .get(Setting.ApplicationTheme)
+        .collectAsStateWithLifecycle()
+
+    val state = rememberPickerState(initialValue = themeValue)
+
+    val valueText = when (themeValue) {
```

Comment:
maybe use Enum here?

Reply:
A little hard to understand, but if this is the only place we will handle the values, it could stay as it is. In case we had to handle the value in multiple (different) places, then we really should change it to enum.